docs/sofapython3/examples/component/constraint/CableConstraint/CableSmoothActuation.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+__authors__ = "tnavez"
+
+import Sofa
+
+import os
+path = os.path.dirname(os.path.abspath(__file__))+'/mesh/'
+
+def createScene(rootNode):
+
+    rootNode.addObject('RequiredPlugin', pluginName= ['SoftRobots', 
+        'Sofa.Component.SolidMechanics.Spring', 
+        'Sofa.Component.Engine.Select', 
+        'Sofa.Component.ODESolver.Backward',
+        'Sofa.Component.IO.Mesh',
+        'Sofa.GL.Component.Rendering2D',
+        'Sofa.GL.Component.Rendering3D',
+        'Sofa.GL.Component.Shader',
+        'Sofa.Component.LinearSolver.Preconditioner',
+        'Sofa.Component.LinearSolver.Iterative',
+        'Sofa.Component.Diffusion',
+        'Sofa.Component.SolidMechanics.FEM.Elastic',
+        'Sofa.Component.LinearSolver.Direct',
+        'Sofa.Component.Topology.Mapping',
+        'Sofa.Component.AnimationLoop',
+        'Sofa.Component.Constraint.Lagrangian.Correction',
+        'Sofa.Component.Constraint.Lagrangian.Solver',
+        'Sofa.Component.Mass',
+        'Sofa.Component.StateContainer',
+        'Sofa.Component.Topology.Container.Constant',
+        'Sofa.Component.Topology.Container.Dynamic',
+        'Sofa.Component.Visual',
+        'Sofa.Component.Mapping.Linear'])
+
+    rootNode.addObject('VisualStyle', displayFlags='showVisualModels hideBehaviorModels showCollisionModels hideBoundingCollisionModels hideForceFields showInteractionForceFields hideWireframe')
+
+    rootNode.addObject('FreeMotionAnimationLoop')
+    rootNode.addObject('GenericConstraintSolver', printLog='0')
+
+    ################################################################################################################
+    ################################################### Bunny ######################################################
+    ################################################################################################################
+    bunny = rootNode.addChild('bunny')
+    bunny.addObject('EulerImplicitSolver', name='odesolver')
+    bunny.addObject('SparseLDLSolver', template='CompressedRowSparseMatrixMat3x3d')
```

Comment:
Could you check if `EigenSimplicialLDLT` is faster than `SparseLDLSolver`? If it is, please replace it

Reply:
Yes it is !